manifests/init.pp

```diff
@@ -87,4 +91,31 @@
       before  => File['satellite_config_yaml'],
     }
   }
+
+  if $trusted_external_command {
+    file { '/etc/puppetlabs/puppet/trusted-external-commands':
+      ensure  => directory,
+      owner   => 'pe-puppet',
+      group   => 'pe-puppet',
```

Comment:
Wouldn't it make sense to have root own this? When would Puppet ever write to this directory at runtime? Feels like a security risk.